theme/theme.inc

```diff
@@ -67,7 +67,9 @@ function islandora_solr_preprocess_islandora_solr(&$variables) {
     // Thumbnail.
     $path = url($result['thumbnail_url'], array('query' => $result['thumbnail_url_params']));
     $image = theme('image', array('path' => $path));
```

Comment:
To avoid potentially stomping on markup generated here (generating markup twice, but only using the last copy), we might want to instead pull out the parameters into a variable... Something like:
```php
$image_params = array(
  'path' => $path,
);
if (isset($result['object_label'])) {
  $image_params['alt'] = $result['object_label'];
}
$image = theme('image', $image_params);
```